internal/cli/kraft/cloud/utils/print.go

```diff
@@ -830,16 +816,11 @@ func PrettyPrintInstance(ctx context.Context, instance *kcinstances.GetResponseI
 		},
 	}
 
-	if fqdn != "" {
-		if strings.HasPrefix(fqdn, "https://") {
-			entries = append(entries, fancymap.FancyMapEntry{
-				Key:   "url",
-				Value: fqdn,
-			})
-		} else {
```

Comment:
shouldn't we still prepend this,?

Reply:
Having this information requires 1 additional API lookup call with the current way things are set up.  Removing is just a simplification and does not hurt UX.